odxtools/utils.py

```diff
@@ -6,7 +6,7 @@
 
 from .odxlink import OdxDocFragment
 
-def read_description_from_odx(et_element: Optional[ElementTree.Element]) \
+def read_description_from_et(et_element: Optional[ElementTree.Element]) \
```

Comment:
Can you split automatic simple updates like changing a function name into their own PR?

That way reviewing the rest of the logic won't be so difficult

Reply:
that can be done